src/api_v2/node.md

```diff
@@ -595,6 +595,109 @@ HTTP/1.1 200 OK
 
       */
 
+== [POST] api/nodes/applyPolicy
+
+    /**
+    @api {post} /api/nodes/{id} 7. Trigger all agent run
+    @apiVersion 8.0.0
+    @apiName applyPolicyAllNodes 
+    @apiGroup Nodes
+
+    @apiParam (Multi) {class}       classes Set class passed as parameter to make it available during agent run.
+
+    @apiDescription This API allows to trigger an agent run on the target node. Response content a json stating if agent has started on each node.
+
+
+    @apiExample Run agent  
+curl -H "X-API-Token: yourToken" -X POST  -H "Content-Type: application/json" https://rudder.example.com/rudder/api/latest/nodes/applyPolicy
+
+    @apiSuccessExample 
+{
+  "action": "applyPolicyAllNodes",
+  "result": "success",
+  "data": [
+    {
+      "id": "edd9a756-ac7e-4820-8e75-83225ebcabec",
+      "hostname": "relay.rudder.local",
+      "result": "Started"
+    },
+    {
+      "id": "root",
+      "hostname": "server.rudder.local",
+      "result": "Started"
+    }
+  ]
+}
+
+      */
+
+
+
+== [POST] api/nodes/{id}/applyPolicy
+
+    /**
+    @api {post} /api/nodes/{id} 8. Trigger an agent run
+    @apiVersion 8.0.0
+    @apiName applyPolicy 
+    @apiGroup Nodes
+
+    @apiUse nodeId
+    @apiParam (Multi) {class}       classes Set class passed as parameter to make it available during agent run.
+
+    @apiDescription This API allows to trigger an agent run on the target node. Response is not a json but the actual agent output on the node.
```

Comment:
This API allows to trigger an agent run on the target node. Response is the streamed agent output directly from the node (and so not in JSON).